c_common/models/live_packet_gather/live_packet_gather.c

```diff
@@ -22,6 +22,12 @@ static circular_buffer without_payload_buffer;
 static circular_buffer with_payload_buffer;
 static bool processing_events = false;
 
+//! Provanence data store
+static uint32_t number_of_over_flows_none_payload = 0;
+static uint32_t number_of_over_flows_payload = 0;
+static uint32_t NUMBER_OF_PROVANENCE_DATA_ELEMENTS = 2;
+static uint32_t SIZE_OF_PROVANENCE_ELEMENTS_IN_BYTES = 8;
```

Comment:
Do you guys mean Provenance?

From: neworderofjamie [mailto:notifications@github.com]
Sent: 29 July 2015 11:07
To: SpiNNakerManchester/SpiNNFrontEndCommon <SpiNNFrontEndCommon@noreply.github.com>
Subject: Re: [SpiNNFrontEndCommon] added some provanence data gathering from the live packet gatherer an… (#14)


In c_common/models/live_packet_gather/live_packet_gather.c<https://github.com/SpiNNakerManchester/SpiNNFrontEndCommon/pull/14#discussion_r35744682>:

> @@ -22,6 +22,12 @@ static circular_buffer without_payload_buffer;

>  static circular_buffer with_payload_buffer;

>  static bool processing_events = false;

>

> +//! Provanence data store

> +static uint32_t number_of_over_flows_none_payload = 0;

> +static uint32_t number_of_over_flows_payload = 0;

> +static uint32_t NUMBER_OF_PROVANENCE_DATA_ELEMENTS = 2;

> +static uint32_t SIZE_OF_PROVANENCE_ELEMENTS_IN_BYTES = 8;

How come the naming of the last two is inconsistent? Also, it might make sense to put these in a struct - Then you wouldn't need the enum as well AND the code to copy it back into SDRAM would be simpler

—
Reply to this email directly or view it on GitHub<https://github.com/SpiNNakerManchester/SpiNNFrontEndCommon/pull/14/files#r35744682>.